### PR TITLE
Improve ns-process-data images when using existing COLMAP model

### PR DESCRIPTION
**Pull request by jkulhanek**
This PR implements the following:
- Additional flag --skip-images for ns-process-data, which skips copying and downscaling of images. The typical use case is if the user already downscaled the images and wants to run just COLMAP
- Additional parameter --colmap-model-path can be used with --skip-colmap. It allows specifying the path to the COLMAP model as opposed to always using "colmap/sparse/0"
- Fixes a bug with --skip-colmap. When --skip-colmap is used with an existing COLMAP model, the filenames are not renamed (but they are renamed when they are copied and downscaled). This makes the resulting transforms.json contain invalid paths.

NOTE: I accidentally closed PR #1366. This PR contains the same changes.

### CODE REVIEW

File: scripts/process_data.py [hunk 1]
```diff
@@ -74,6 +74,10 @@ class ProcessImages:
         will downscale the images by 2x, 4x, and 8x."""
     skip_colmap: bool = False
     """If True, skips COLMAP and generates transforms.json if possible."""
+    skip_images: bool = False
```
Comment (tancik) [inline scripts/process_data.py:77]: Make more descriptive, -> `skip_image_processing`

File: scripts/process_data.py [hunk 2]
```diff
@@ -133,11 +150,16 @@ def main(self) -> None:
                 sys.exit(1)
 
         # Save transforms.json
-        if (colmap_dir / "sparse" / "0" / "cameras.bin").exists():
+        if self.skip_colmap:
+            colmap_model_path = Path(self.colmap_model_path)
+        else:
+            colmap_model_path = colmap_dir / "sparse" / "0"
+
+        if (colmap_model_path / "cameras.bin").exists():
```
Comment (tancik) [inline scripts/process_data.py:158]: We should print an error if the file doesn't exist.

**Review by tancik — COMMENTED** [review]
Thanks for cleaning up this pipeline. We have received multiple requests for supporting externally colmaped data.

File: scripts/process_data.py (hunk 1 above)
Comment (jkulhanek) [reply scripts/process_data.py]: Ok

File: scripts/process_data.py (hunk 2 above)
Comment (jkulhanek) [reply scripts/process_data.py:142]: I added error printing. Is this what you meant?

**Review by tancik — APPROVED** [review]
LGTM

**Pull request merged**